api/db/seeds/data/common/common-builder.js

```diff
@@ -1,6 +1,6 @@
 import { PIX_ADMIN } from '../../../../lib/domain/constants.js';
-
-const { ROLES } = PIX_ADMIN;
+import { badges } from '../../../constants';
```

Comment:
```suggestion
import { badges } from '../../../constants.js';
```

<img width="497" alt="Capture d’écran 2023-06-09 à 11 42 59" src="https://github.com/1024pix/pix/assets/58915422/a4782a7b-749d-4a24-8cff-ad52da7bebf4">